gitops/AzureDevOps/README.md

```diff
@@ -0,0 +1,202 @@
+# Approach
+
+Taking into account that devops-as-code provides convenient wrapper for *NIX and Windows systems, it is very easy to using together with different CI tools.
+
+It is very easy to integrate devops-as-code into your git flow in Azure DevOps you should do the following:
+
+1) Add wrappers to your project. You can either copy/paste them from another project, or execute `xl wrapper` command in the root of your project to generate them.
+2) Define devops-as-code YAML in your project's directory.  
+3) Define different pipelines depending on the OS. On Windows you should use a `script` step or a `bash` step in your `azure-pipeline.yml` file and in there you
```

Comment:
Eeeem this is a bit miss-leading. First of all, can you re-phrase this sentence? It is too long and complicated. I had to read it through few times and still didn't get it :(

Beside that, i didn't really understand what should i use for Windows: 'script' or 'bash'?